cmd/vicadmin/server.go

```diff
@@ -551,6 +548,10 @@ func (s *server) index(res http.ResponseWriter, req *http.Request) {
 	defer trace.End(trace.Begin(""))
 	ctx := context.Background()
 	sess, err := s.getSessionFromRequest(ctx, req)
+	if err != nil {
+		http.Redirect(res, req, "/logout", http.StatusTemporaryRedirect)
+		return
```

Comment:
My understanding is that if `getSessionFromRequest` returns an error here, the `if sess == nil` block below will no longer get executed and the degraded dashboard will not kick in. Just wanted to check if that's the intended behavior or not.

Also, while doing a quick test with your branch,  if I set `sess = nil` explicitly right below the `sess, err := s.getSessionFromRequest(ctx, req)` line, I get too many redirects on the browser. I'll dig deeper to see what's up.